python/aswfdocker/builder.py

```diff
@@ -184,95 +184,116 @@ def _get_conan_base_cmd(self, version_info):
         base_cmd.append(tag)
         return base_cmd
 
+    def _build_conan_package(
+        self,
+        image,
+        version,
+        dry_run,
+        keep_source,
+        keep_build,
+        conan_login,
+        build_missing,
+    ):
+        major_version = utils.get_major_version(version)
+        version_info = self.index.version_info(major_version)
+        base_cmd = self._get_conan_base_cmd(version_info)
+        if conan_login:
+            self._run_in_docker(
+                base_cmd,
+                [
+                    "conan",
+                    "user",
+                    "-p",
+                    "-r",
+                    self.build_info.docker_org,
+                ],
+                dry_run,
+            )
+        self._run_in_docker(
+            base_cmd,
+            [
+                "conan",
+                "config",
+                "set",
+                f"general.default_profile={version_info.conan_profile}",
+            ],
+            dry_run,
+        )
+        full_version = version_info.package_versions.get(
+            "ASWF_" + image.upper() + "_VERSION"
+        )
+        conan_version = (
+            f"{image}/{full_version}"
+            f"@{self.build_info.docker_org}/{version_info.conan_profile}"
+        )
+        build_cmd = [
+            "conan",
+            "create",
+            f"/tmp/c/recipes/{image}",
```

Comment:
python/aswfdocker/builder.py defines CONAN_USER_HOME as "/tmp/c", could this be referenced here to avoid hard coding the output path of "conan create", or is this being used in a different context?
Also, would it make sense to use the Python `tempfile` module to avoid UNIX-specific temp paths?

Reply:
Good point, I'll add this as a constant!